lib/info.js

```diff
@@ -0,0 +1,44 @@
+const axios = require('axios')
+const urljoin = require('url-join')
+const {getMetadata} = require('./get.js')
+const {customMarked} = require('./utils/marked.js')
+const {logger} = require('./utils/log-handler.js')
+const utils = require('../lib/utils/common')
+
+const getInfo = async (dhpkgid) => {
+  const { publisher, pkg } = utils.parseIdentifier(dhpkgid)
+
+  const metadata = await getMetadata(publisher, pkg)
+  const readme = await getReadme(urljoin(metadata.bitstore_url, 'README.md'))
+
+  const firstParagraphReadme = `# ABOUT\n${readme.substring(0, 200)}... *see more below*`
```

Comment:
first 200? what if in the middle of the word? maybe 
`readme.substring(0, 200).split(' ').pop().join(' ') ...` would be better